Makefile.inc

```diff
@@ -8,6 +8,8 @@ DOCKER_RUN_FLAGS = \
 	--rm \
 	--name "$(CONTAINER_NAME)" \
 	-p 8888:8888 \
+        --network host \
+	-v /tmp:/tmp \
```

Comment:
Could we do this part by instead adding the following to `clipper/Makefile`?
```
DOCKER_RUN_FLAGS += --network host -v /tmp:/tmp
```

Also, do we really need `-v /tmp:/tmp`, or is that an artifact from a merge conflict? (cf. 54be3a3d6f7f254b145788b8082ea8013b07fb45)